components/reusable-components/reusable-modal/ReusableModal.tsx

```diff
@@ -67,15 +84,15 @@ const ReusableModal = ({ title, description, isOpen, onClose = () => {} }: Reusa
           <div className={style.modalButtons}>
             <Button
               type="button"
-              buttonText="Cancel"
-              buttonClass={['primaryButton']}
-              onClick={onClose}
+              buttonText={primaryButtonLabel || 'Proceed'}
```

Comment:
you already have added a default value for primaryButtonLabel, no need to add it as a fallback here as well